### PR TITLE
Fix/nft type

### PR DESCRIPTION
**Pull request by trust0212**

### CODE REVIEW

File: src/eth/helpers.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ const areUrlExtensionsSupportedForType = (
 };
 
 const isAssetVideo = (asset: OpenSeaAssetExtended | NftPortAssetExtended) => {
-  return areUrlExtensionsSupportedForType(asset, SUPPORTED_VIDEO_EXTENSIONS);
+  return asset.animation_url || asset.animation_original_url;
```
Comment (chinu-dev) [inline src/eth/helpers.ts:62]: i would delete 
const isAssetGif
completely if we dont want it

File: src/eth/helpers.ts [hunk 2]
```diff
@@ -319,8 +294,51 @@ export const nftportAssetToCollectible = async (
       }
 
       videoUrl = [animation_url, animation_original_url, ...imageUrls].find(
-        url => url && SUPPORTED_VIDEO_EXTENSIONS.some(ext => url.endsWith(ext)),
+        url => !!url
       )! ?? null;
+
+      // Determine if animation is video, html or audio
+      const res = await fetch(videoUrl, { method: 'HEAD' });
+      const _isVideo = res.headers.get('Content-Type')?.includes('video');
```
Comment (chinu-dev) [inline src/eth/helpers.ts:302]: i think ask about content type from response headers is better way then ask for file extension, or do both ways. some of them will not use extension but only IPFS CID.

File: src/eth/helpers.ts (hunk 1 above)
Comment (chinu-dev) [reply src/eth/helpers.ts:62]: i see you switched the order. i guess if @sacred-danny is for it we can have GIF still for now, if there is gifUrl in our model. 

Is it really necessary to have it splitted?

File: src/eth/helpers.ts [hunk 3]
```diff
@@ -319,8 +294,51 @@ export const nftportAssetToCollectible = async (
...
+          gifUrl = frameUrl;
+          // frame url for the gif is computed later in the collectibles page
+          frameUrl = null;
+        }
+      }
+    } else if (isAssetGif(asset)) {
+      mediaType = 'GIF';
+      // frame url for the gif is computed later in the collectibles page
+      frameUrl = null;
+      gifUrl = imageUrls.find(url => url?.endsWith('.gif'))! ?? null;
```
Comment (chinu-dev) [inline src/eth/helpers.ts:341]: still here i want both extension + http header to be "image/gif"

File: src/eth/helpers.ts [hunk 4]
```diff
@@ -319,8 +294,51 @@ export const nftportAssetToCollectible = async (
...
+      } else if (_isHtml) {
+        mediaType = 'HTML';
+      }
+
+      imageUrl = imageUrls.find(url => !!url)! ?? null;
+
+    }  else if (isAssetThreeDAndIncludesImage(asset)) {
+      mediaType = 'THREE_D';
+      threeDUrl = [animation_url, animation_original_url, ...imageUrls].find(
+        url => url && SUPPORTED_3D_EXTENSIONS.some(ext => url.endsWith(ext)),
```
Comment (chinu-dev) [inline src/eth/helpers.ts:319]: plus we should do url.toLowerCase().endsWith(ext.toLowerCase())

**Comment by chinu-dev** [comment]
Please do not commit any changes to examples project. why you are deleting it. can you create different commit where it will be clean?

**Comment by trust0212** [comment]
I already committed with example directory yesterday.

**Comment by trust0212** [comment]
@chinu-dev @sacred-danny 
image

**Pull request closed without merging**